keras_cv/layers/preprocessing/base_image_augmentation_layer.py

```diff
@@ -258,7 +258,8 @@ def _augment(self, inputs):
                 image=image,
             )
             result[BOUNDING_BOXES] = bounding_boxes
-        return result
+        inputs.update(result)  # preserve unmodified, additional entries.
+        return inputs
```

Comment:
Should we use the result dict instead? if user change the inputs dict down the stream, does it affect the result dict returned as well

Reply:
Good question. I did not check this variant, but decided to modify the code anyway to return `results` - I feel like modifying `inputs`, returning `inputs` and at the same time, having them unmodified is counterintuitive, and too surprising.